pwiz_tools/commandline/msconvert.cpp

```diff
@@ -400,98 +471,57 @@ Config parseCommandLine(int argc, char** argv)
-              << "msconvert data.RAW --filter \"msLevel 2-\"\n"
-              << endl
-              << "# apply ETD precursor mass filter\n"
-              << "msconvert data.RAW --filter ETDFilter\n"
-              << endl
-              << "# remove non-flanking zero value samples\n"
-              << "msconvert data.RAW --filter \"zeroSamples removeExtra\"\n"
-              << endl
-              << "# remove non-flanking zero value samples in MS2 and MS3 only\n"
-              << "msconvert data.RAW --filter \"zeroSamples removeExtra 2 3\"\n"
-              << endl
-              << "# add missing zero value samples (with 5 flanking zeros) in MS2 and MS3 only\n"
-              << "msconvert data.RAW --filter \"zeroSamples addMissing=5 2 3\"\n"
-              << endl
-              << "# keep only HCD spectra from a decision tree data file\n"
-              << "msconvert data.RAW --filter \"activation HCD\"\n"
-              << endl
-              << "# keep the top 42 peaks or samples (depending on whether spectra are centroid or profile):\n"
-              << "msconvert data.RAW --filter \"threshold count 42 most-intense\"\n"
-              << endl
-              << "# multiple filters: select scan numbers and recalculate precursors\n"
-              << "msconvert data.RAW --filter \"scanNumber [500,1000]\" --filter \"precursorRecalculation\"\n"
-              << endl
-              << "# multiple filters: apply peak picking and then keep the bottom 100 peaks:\n"
-              << "msconvert data.RAW --filter \"peakPicking true 1-\" --filter \"threshold count 100 least-intense\"\n"
-              << endl
-              << "# multiple filters: apply peak picking and then keep all peaks that are at least 50% of the intensity of the base peak:\n"
-              << "msconvert data.RAW --filter \"peakPicking true 1-\" --filter \"threshold bpi-relative .5 most-intense\"\n"
-              << endl
-              << "# use a configuration file\n"
-              << "msconvert data.RAW -c config.txt\n"
-              << endl
-              << "# example configuration file\n"
-              << "mzXML=true\n"
-              << "zlib=true\n"
-              << "filter=\"index [3,7]\"\n"
-              << "filter=\"precursorRecalculation\"\n"
-              << endl
-              << endl;
+        ShowExamples(usage);
     }
     else
     {
         // append options description to usage string
         usage << od_config;
 
         // extra usage
-        usage << SpectrumListFactory::usage(detailedHelp, "(run this program with --help to see details for all filters)", consoleBounds.first);
-        usage << ChromatogramListFactory::usage(detailedHelp, nullptr, consoleBounds.first) << endl;
+        usage << SpectrumListFactory::usage(detailedHelp, "(run this program with --help to see details for all filters)", wrapWidth);
+        usage << ChromatogramListFactory::usage(detailedHelp, nullptr, wrapWidth) << endl;
```

Comment:
This wrapWidth should be unlimited for --doc so instead of:
```
Many filters take 'int_set' arguments.  An 'int_set' is a list of intervals of
the form [a,b] or a[-][b].
For example '[0,3]' and '0-3' both mean 'the set of integers from 0 to 3
inclusive'.
'1-' means 'the set of integers from 1 to the largest allowable number'.  
'9' is also an integer set, equivalent to '[9,9]'.
'[0,2] 5-7' is the set '0 1 2 5 6 7'. 
```

We get:
```
Many filters take 'int_set' arguments.  An 'int_set' is a list of intervals of the form [a,b] or a[-][b].
For example '[0,3]' and '0-3' both mean 'the set of integers from 0 to 3 inclusive'.
'1-' means 'the set of integers from 1 to the largest allowable number'.
'9' is also an integer set, equivalent to '[9,9]'.
'[0,2] 5-7' is the set '0 1 2 5 6 7'.
```

Reply:
OK, as previously suggested it's now using the default width for the first section to get the nice columization, and a crazy wide width for the rest so the viewer can wrap as appropriate.